mobile_verifier/src/reward_shares.rs

```diff
@@ -206,9 +212,19 @@ impl MapperShares {
         // however the function is setup to allow the verification mapper shares to be easily
         // added without impacting code structure ( the per share value for those will be different )
         let total_mapper_shares = discovery_mappers_count * DISCOVERY_MAPPING_SHARES;
```

Comment:
```suggestion
        let total_disco_mapper_shares = discovery_mappers_count * DISCOVERY_MAPPING_SHARES;
```

Reply:
Still think this is relevant, line 217 is not calculating the total_mapper_shares ( ie total shares for mapping ) but instead is calculating the total discovery mapping shares.  Keeping it as total_mapper_shares is a misleading var name

Reply:
Sorry I think it got lost in my rebase